src/no/runsafe/UserControl/database/PlayerDatabase.java

```diff
@@ -64,6 +65,40 @@ public ISchemaUpdate getSchemaUpdateQueries()
 
 		update.addQueries("ALTER TABLE player_db ADD COLUMN temp_ban datetime NULL");
 		update.addQueries("ALTER TABLE player_db ADD COLUMN uuid VARCHAR(255) NULL");
+
+		update.addQueries(
+			// Add column for banner UUID
+			"ALTER TABLE player_db ADD COLUMN ban_by_uuid VARCHAR(36) NULL;",
+			// Convert banner usernames to UUIDs
+			"UPDATE IGNORE player_db SET `ban_by_uuid` = " +
+				"(SELECT `uuid` FROM player_db WHERE `name`=`player_db`.`ban_by`) " +
+				"WHERE `ban_by` != 'console' OR `ban_by` IS NOT NULL`;",
+			// Convert console bans to UUIDs.
+			"UPDATE IGNORE player_db SET `ban_by_uuid` = '00000000-0000-0000-0000-000000000000'" +

```

Comment:
We cound insert a "player" with this uuid named console in the table and the ban_by match would work - would also maybe be beneficial in other exciting ways..